nbs/02_template.ipynb

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
_Commented on notebook `nbs/02_template.ipynb` Cell 4 Line 21_
```python
)
from fastcore.script import (
    call_parse,
)  # for @call_parse, https://fastcore.fast.ai/script
import json  # for nicely printing json and yaml

#import functions from core module (optional, but most likely needed). 
from $PACKAGE_NAME import(
```
I don't think the "$PACKAGE_NAME" variable will be recognised by python. This might have to be replaced manually by the author.

Reply:
This doesn't actually go in but is modified by the augment shell script 
`sed "s/\$PACKAGE_NAME/$GIT_REPO_NAME/g" $NBDEV_PROJECT_FOLDER/nbs/02_template.ipynb >$NBDEV_PROJECT_FOLDER/nbs/02_template.ipynb.tmp`

Reply:
Yes, I got this after I had written the comment. Tried to remove the comment but apparently not successfully..